src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1SubmissionService.java

```diff
@@ -115,7 +116,12 @@ private DocumentTypeItem createEt1DocumentType(CaseDetails caseDetails, String u
 
     public DocumentInfo createEt1(CaseDetails caseDetails, String userToken, String pdfSource)
             throws PdfServiceException {
-        byte[] pdf = pdfService.convertCaseToPdf(caseDetails.getCaseData(), pdfSource);
+        byte[] pdf = pdfService.convertCaseToPdf(
+                caseDetails.getCaseData(),
+                pdfSource,
+                ET1,
+                "representative",
```

Comment:
Minor - This code is used by both the citizen claimant and the representative so whilst there's no functional change on this value, it might be better reflected as claimantparty
```suggestion
                "claimantparty",
```